lib/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ cmake_minimum_required(VERSION 3.4.1)
 
 project (LiteCoreJNI)
 
-set(LITE_CORE_ROOT ${PROJECT_SOURCE_DIR}/../couchbase-lite-core)
+set(LITE_CORE_ROOT ${PROJECT_SOURCE_DIR}/../../couchbase-lite-core)
```

Comment:
This is fine as it is, at least for now.  I wonder, though, if it should be passed in.  It really should be the root build that knows where Core is located...